tests/phpunit/tests/l10n/getLocale.php

```diff
@@ -40,10 +40,6 @@ public function test_local_option_should_take_precedence_on_multisite() {
 	 * @group ms-required
 	 */
 	public function test_network_option_should_be_fallback_on_multisite() {
-		if ( ! is_multisite() ) {
-			$this->markTestSkipped( 'This test requires Multisite.' );
-		}
```

Comment:
`@group` tags are all nice and dandy, but don't prevent a test from _not_ being run without complying with certain conditions. Only for running them selectively.

This condition should not be removed.